src/HLSLTree.h

```diff
@@ -328,9 +329,15 @@ struct HLSLStatement : public HLSLNode
 struct HLSLAttribute : public HLSLNode
 {
     static const HLSLNodeType s_type = HLSLNodeType_Attribute;
-    HLSLAttributeType   attributeType = HLSLAttributeType_Unknown;
```

Comment:
Do you have anything against using C++11 member initialization? Do you have to support old compilers?